elizabeth/data/da/personal.json

```diff
@@ -2022,5 +2022,36 @@
 		"Test"
 	  ]
 	}
+  },
+  "personality": {
+	  "mbti": [
+		  "ISFJ",
+		  "ISTJ",
+		  "INFJ",
+		  "INTJ",
+		  "ISTP",
+		  "ISFP",
+		  "INFP",
+		  "INTP",
+		  "ESTP",
+		  "ESFP",
+		  "ENFP",
+		  "ENTP",
+		  "ESTJ",
+		  "ESFJ",
+		  "ENFJ",
+		  "ENTJ"
+	  ],
+	  "rheti": [
```

Comment:
Or you can move it to specific data providers. Class `USA` in elizabeth.builtins.